README.md

```diff
@@ -31,8 +31,11 @@ compiler such as GNU GCC, the CMake build system and, optionally, zlib.
 
 ```bash
 $ git clone https://github.com/Equinor/libecl
-$ cmake libecl
-$ cmake --build libecl --target install
+$ mkdir libecl/build
+$ cd libecl/build
+$ cmake ..
+$ make
+$ sudo make install
```

Comment:
Why are we recommending to install using `sudo`? That sounds like horrible advice one would only find on the internet...

Reply:
What would you suggest as an alternative? The «nice» way to do this would be to provide ‘.deb’ and ‘.rpm’. CMake has tools to generate these.

Reply:
I actually checked now and our CMake Python stuff is so incorrect that `sudo make install` doesn’t do the correct thing.

`CMAKE_INSTALL_PREFIX` is a pretty advanced feature, how would you suggest they use it? Install into a virtualenv?

Reply:
Either into a virtualenv or into an install prefix that they append to their paths. The next sentence in the readme goes:

> Then, you must tell Python where to find the package[1]:

I think most users will be happy with pip-installing it. And the ones who need the C-API will probably be fine with dealing with `LD_LIBRARY_PATH`?

Reply:
I’m not comfortable asking users who need these instructions to mess around with environment variables. Very user unfriendly. The correct place for user-managed software is `/usr/local`, which is the default for `make install`, but requires root.

I’ll ping @jondequinor for his opinion on this. Maybe also @jokva .

Reply:
I kind of like how `segyio` has done it:
https://github.com/equinor/segyio#build-segyio

Reply:
I think `sudo` is superfluous, and not often seen in build instructions. I think the instructions should assume as little as possible about the user's environment. So the assumption that the user is going to install into some pre-existing prefix is fine by me. Setting `LD_LIBRARY_PATH` is not good in my opinion: unsafe and unnecessary.